api/Worklet.json

```diff
@@ -20,10 +20,26 @@
             "version_added": false
           },
           "firefox": {
-            "version_added": true
+            "version_added": "52",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "dom.worklet.enabled",
+                "value_to_set": "true"
+              }
+            ],
+            "notes": "Available only in Firefox Nightly."
```

Comment:
I'd get rid of this note, as it is a bit confusing, IMO. You could read this as "available by default in Nightly, but behind a pref in non-Nightly versions", or "Available behind a pref, but only in Nightly". I've tested it, and the truth is that it is available in all versions, but behind a pref. And it is one of those prefs that doesn't appear by default; you have to set it yourself. I'm not sure if we have any convention for setting those @Elchi3?

Reply:
When I looked at the code which is part of the first bug (and the code on the tip of mozilla-central), the `MOZ_CRASH` macro is used to prevent the construction of Worklet objects on non-Nightly builds (unless that changed recently).

Reply:
Hrm, so The Worklet object is available in non-Nightly versions, but you just can't use it for anything?